imagebuildah/stage_executor.go

```diff
@@ -1164,6 +1173,24 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 		// determining if a cached layer with the same build args already exists
 		// and that is done in the if block below.
 		if checkForLayers && step.Command != "arg" && !(s.executor.squash && lastInstruction && lastStage) {
+			// Is true if current STEP is `COPY` or `ADD`, denoting that
+			// cache lookup can be attempted only after performing a RUN
+			canMatchCacheOnlyAfterRun = (step.Command == command.Add ||
+				step.Command == command.Copy)
```

Comment:
```suggestion
			canMatchCacheOnlyAfterRun = (step.Command == command.Add || step.Command == command.Copy)
```

Reply:
hmm.. i committed this change and `gofmt` reverted everything back to online weird.